native/cocos/platform/ios/modules/Accelerometer.mm

```diff
@@ -145,9 +145,15 @@ - (void)setMotionInterval:(float)interval {
 
 namespace cc {
 void Accelerometer::setAccelerometerEnabled(bool isEnabled) {
+#if !defined(CC_TARGET_OS_TVOS)
```

Comment:
CC_TARGET_OS_TVOS is not need. Engine doesn't support tvOS.

Reply:
So should I remove all CC_TARGET_OS_TVOS related code in engine's code

Reply:
Yep, can remove it.